tests/Cms/App/AppComponentsTest.php

```diff
@@ -76,6 +76,46 @@ public function testJsPlugin()
         $this->assertEquals($expected, js('something.js'));
     }
 
+    public function testKirbytext()
+    {
+        $text     = 'Test';
+        $expected = '<p>Test</p>';
+
+        $this->assertEquals($expected, $this->kirby->kirbytext($text));
+    }
+
+    public function testKirbytextWithSafeMode()
+    {
+        $text     = '<h1>Test</h1>';
+        $expected = '&lt;h1&gt;Test&lt;/h1&gt;';
```

Comment:
To make sure that Markdown is still parsed:

```suggestion
        $text     = '<h1>**Test**</h1>';
        $expected = '&lt;h1&gt;<strong>Test</strong>&lt;/h1&gt;';
```